Jenkinsfile

```diff
@@ -96,6 +96,40 @@ pipeline {
           }
         } // Linux builds and tests
 
+        stage('Windows builds') {
+          agent {
+            label 'windows10&&unified'
+          }
+          stages {
+            stage('Build') {
+              steps {
+                runningOn(env.NODE_NAME)
+                dir('lib_xcore_math') {
+                  checkout scm
+                  // fetch submodules
+                  bat 'git submodule update --init --recursive --jobs 4'
+                  withTools(params.TOOLS_VERSION) {
+                    withVS {
```

Comment:
not sure why you need `withVS` here as it doesn't build `x86`

Reply:
I don't know either, but if I remove that command I get the error below. @BrennanGit, is this behaviour expected?

```
[2023-09-28T09:32:34.311Z] C:\jenkins\workspace\XMOS_lib_xcore_math_PR-152\lib_xcore_math>cmake -B build_xs3a -DXMATH_SMOKE_TEST=${params.XMATH_SMOKE_TEST} --toolchain=etc/xmos_cmake_toolchain/xs3a.cmake -G"Ninja" 
[2023-09-28T09:32:36.825Z] -- Found Python3: C:/Python310/python3.exe (found version "3.10.7") found components: Interpreter 
[2023-09-28T09:32:36.825Z] CMake Error: CMake was unable to find a build program corresponding to "Ninja".  CMAKE_MAKE_PROGRAM is not set.  You probably need to select a different build tool.
[2023-09-28T09:32:36.825Z] -- Configuring incomplete, errors occurred!
[2023-09-28T09:32:36.825Z] See also "C:/jenkins/workspace/XMOS_lib_xcore_math_PR-152/lib_xcore_math/build_xs3a/CMakeFiles/CMakeOutput.log".
script returned exit code 1
```

Reply:
I had to check but it looks like ninja is installed on windows as part of the visual studio build tools.